### ISSUE TITLE
Android v3 and IOS v4 sdk for PSD2 Strong Customer Authentication

### ISSUE THREAD

**Issue by crisp978** [Taracque/cordova-plugin-braintree#70]
Someone has already begun to integrate the Android v3 and IOS v4 sdk into the plugin?

**Comment by Deeeej** [comment]
Yes, the deadline in Europe for SCA (Strong customer authentication) which basically mandates the use of 3DS - 3d secure, is the 14th September 2019.
This means, unfortunately, the current version of this plugin will result in a much higher level of rejected transactions as there is no method for using the 3DS flows.

Are there any plans to update the plugin to make it work after 14th September 2019?

Here are some articles on the changes: 
https://developers.braintreepayments.com/guides/3d-secure/overview
https://www.braintreepayments.com/blog/psd2-sca-requirements-a-closer-look-at-exemptions/

**Comment by Deeeej** [comment]
On closer inspection it looks like the plugin already supports 3DS 2.0 for iOS as that was released in version 4.10.0 of the SDK and this plugin uses version 4.14.0, see: https://github.com/braintree/braintree_ios/releases?after=4.16.0

But the Android SDK only got support after version 3.3.30, see https://github.com/braintree/braintree_android/releases, which was only released 25 days ago.

**Comment by DevHamburg** [comment]
Are there any plans to update the plugin to make it work after 14th September 2019?

This is very important

**Comment by Samyssmile** [comment]
I received some E-Mails from Braintree, I need to update my plugin to Android v3. Any ideas about progress guys ?

**Comment by Samyssmile** [comment]
> Someone has already begun to integrate the Android v3 and IOS v4 sdk into the plugin?

Did you solved it?

**Comment by jamesdixon** [comment]
@Deeeej or anyone else here have luck with 3ds support?
